Tests/MongoSwiftSyncTests/SyncMongoClientTests.swift

```diff
@@ -320,4 +320,34 @@ final class SyncMongoClientTests: MongoSwiftTestCase {
         expect(try MongoClient("mongodb://localhost:12345", options: options))
             .to(throwError(errorType: MongoError.InvalidArgumentError.self))
     }
+
+    func testConnectionTimeout() throws {
+        let setupClient = try MongoClient.makeTestClient()
+        guard try setupClient.supportsBlockTime() else {
+            printSkipMessage(testName: self.name, reason: "blockTime not supported")
+            return
+        }
+        guard try setupClient.supportsFailCommand() else {
```

Comment:
Supporting blockTime on fail command implies support for failCommand, so this check can be removed now.

Reply:
done